src/docfx/build/Build.cs

```diff
@@ -55,20 +62,20 @@ private static async Task<bool> BuildDocset(string docsetPath, string outputPath
 
                 // run build based on docsets
                 outputPath ??= Path.Combine(docsetPath, docset.Config.Output.Path);
-                await Run(docset, fallbackDocset, options, errorLog, outputPath, input, repositoryProvider, localizationProvider);
+                Run(docset, fallbackDocset, options, errorLog, outputPath, input, repositoryProvider, localizationProvider).GetAwaiter().GetResult();
```

Comment:
what do we gain by changing the method chain to synchronous instead of asynchronous all the way?

Reply:
`await Task.WhenAll` wasn't true parallel.

Reply:
can we use `ParallelUtility.ForEach` to not use `.GetAwaiter().GetResult()`?

Reply:
split it to a separate PR.